scripts/qa_crowbarsetup.sh

```diff
@@ -985,6 +985,17 @@ EOF
         rm -f $pfile
         crowbar ntp proposal commit default
     fi
+    if [[ $keep_existing_hostname = 1 ]] ; then
```

Comment:
i would still add provisioner to `custom_configuration` and just call `update_one_proposal provisioner default` from here and use the existing mechanisms. can you test this?